frontend/server/src/test/java/org/pytorch/serve/device/utils/CudaUtilTest.java

```diff
@@ -0,0 +1,132 @@
+package org.pytorch.serve.device.utils;
+
+import java.util.ArrayList;
+import java.util.LinkedHashSet;
+import org.pytorch.serve.device.Accelerator;
+import org.pytorch.serve.device.AcceleratorVendor;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class CudaUtilTest {
+
+    private CudaUtil cudaUtil = new CudaUtil();
+
+    @Test
+    public void testGetGpuEnvVariableName() {
+        Assert.assertEquals(cudaUtil.getGpuEnvVariableName(), "CUDA_VISIBLE_DEVICES");
+    }
+
+    @Test
+    public void testGetUtilizationSmiCommand() {
+        String[] expectedCommand = {
+            "nvidia-smi",
+            "--query-gpu=index,gpu_name,utilization.gpu,utilization.memory,memory.used",
+            "--format=csv,nounits"
+        };
+        Assert.assertEquals(cudaUtil.getUtilizationSmiCommand(), expectedCommand);
+    }
+
+    @Test
+    public void testSmiOutputToUpdatedAccelerators() {
```

Comment:
Are these entirely new unit tests we are adding?

Reply:
yes. We introduced a new package `org.pytorch.serve.device`. These are unit tests for one of the classes